### DISCUSSION TITLE
How to disable permission on role to specific user?

### DISCUSSION THREAD

**Discussion by donnyatspe** [spatie/laravel-permission#2600] [Q&A]
I have role "Manage XYZ", and this role has permissions like "create", "view", "update", "delete". This will add records into "role_has_permissions" table. And this role we assign to user "User A", "User B" and "User C". This will add records into "model_has_roles" table. If I want add new permission "publish" for "User C" only. So it will add new record into "model_has_permissions" table. If I want disable "create" for "User A" only, it is possible?

**Comment by parallels999** [comment]
Always read other threads first, before opening a new one
https://github.com/spatie/laravel-permission/discussions/1895#discussioncomment-1523163
https://github.com/spatie/laravel-permission/pull/1936
